crates/swc_plugin_runner/src/lib.rs

```diff
@@ -112,8 +141,15 @@ pub fn apply_js_plugin(
     program: Program,
 ) -> Result<Program, Error> {
     (|| -> Result<_, Error> {
-        let _instance = load_plugin(path, cache)?;
-        // TODO: actually apply transform from plugin
+        let instance = load_plugin(path, cache)?;
+
+        let plugin_process = instance
+            .exports
+            .get_native_function::<(i32, u32), i32>("process")?;
+
+        let (alloc_ptr, len) = copy_memory_to_instance(&instance, &program)?;
+        plugin_process.call(alloc_ptr, len)?;
```

Comment:
I think we need to call `free` after this?

Reply:
yes eventually. I left it intentionally for now - first would like to verify passing / transform with serialization first and also want to check bit more if alloc is needed in this form.